pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -11877,4 +11877,8 @@ If you do not have the original file, you may build the library with embedded sp
   <data name="SkylineWindow_submitErrorReportMenuItem_Click_Submitting_an_unhandled_error_report" xml:space="preserve">
 	  <value>Submitting an unhandled error report</value>
   </data>
+  <data name="Colon" xml:space="preserve">
+	  <value>:</value>

```

Comment:
This should be {0}: {1}

Chinese and Japanese will drop the space. The Asian colon is a "double-wide character".

I think you can also add the translated values in this PR.

Reply:
This seems redundant with Skyline.resx:Colon